web/styles/portico/landing_page.css

```diff
@@ -2810,8 +2805,9 @@ button {
                 flex-direction: column;
 
                 .feature-box {
-                    width: 75vw !important;
-                    height: calc(600px - 35vw);
+                    width: 70vw;
+                    height: auto;
```

Comment:
I realized we don't need to have same height for all the boxes when they are all stacked vertically on mobile. So, I changed them to just fit content.